packages/esm-patient-test-results-app/src/panel-view/types.ts

```diff
@@ -90,15 +100,15 @@ export interface ConceptMeta {
   lowAbsolute: number;
   lowCritical: number;
   units: string;
-  getInterpretation: (value: number) => OBSERVATION_INTERPRETATION;
+  getInterpretation: (value: string) => OBSERVATION_INTERPRETATION;
```

Comment:
Are you sure this is not a breaking change elsewhere? maybe you should expect number or string so that it doesn't break when it comes as a number.

Reply:
No, I have verified it thoroughly and this is not breaking. If you see the `getObservationValue`, that is returning a string altogether, and hence I changed the type of the value with string. 

If you scroll down a few lines, I have a `isNaN` check in place so that it doesn't break.
Thanks!